pkg/authz/objecttype/service.go

```diff
@@ -19,22 +19,32 @@ import (
 
 	"github.com/warrant-dev/warrant/pkg/event"
 	"github.com/warrant-dev/warrant/pkg/service"
+	"github.com/warrant-dev/warrant/pkg/wookie"
 )
 
 const ResourceTypeObjectType = "object-type"
 
 type ObjectTypeMap struct {
-	objectTypes map[string]ObjectTypeSpec
+	ObjectTypes map[string]ObjectTypeSpec
```

Comment:
Why does this need to be exported? Is it for the interface?

Reply:
Yes, this allows for `ObjectTypes` to be used for services implementing the ObjectType Service interface, specifically for the `GetTypeMap` method

Reply:
Can you actually update the `ObjectTypeMap` struct to alias the `map[string]ObjectTypeSpec` type?

```go
type ObjectTypeMap map[string]ObjectTypeSpec
```

This will allow callers to use the underlying map (e.g. iterating over it, etc.) if needed. Seems like that might be your intention here @stanleyphu?